.travis.yml

```diff
@@ -41,8 +41,8 @@ notifications:
     on_success: change  # options: [always|never|change] default: always
     on_failure: always  # options: [always|never|change] default: always
     on_start: never     # options: [always|never|change] default: always
-cache:
-  apt: true
-  directories:
-  - node_modules/
-  - public/lib/
+# cache:
```

Comment:
I know. I just removed it temporarily in this PR to make sure that it would still pass Travis.